src/client/components/lazy-chat/lazy-chat.vue

```diff
@@ -74,4 +74,62 @@ export default {
 .lazy-chat__prompt {
   box-shadow: var(--shadow-wide-grey);
 }
+
+.lazy-chat__open-button {
+  width: 54px;
+  height: 54px;
+  border-radius: 50%;
+  padding: 0;
+  margin-right: -1px;
+  margin-bottom: -1px;
+  position: relative;
+}
+
+.lazy-chat__open-button:before {
+  content: '';
+  display: inline-block;
+  margin-left: 0;
+  margin-bottom: -6px;
+  width: 32px;
+  height: 26px;
+  background-image: url("/images/chat-bubble.svg");
+  background-position: center !important;
+  background-size: contain !important;
+  background-repeat: no-repeat !important;
+}
+
+.loading:before {
```

Comment:
Can you change this into a bem-like syntax? I'm not sure styling is scoped, but even if it is, you should follow coding conventions for this project (which would be `lazy-chat__open-button--loading`)

Same goes for the spinner animation, I suppose. Can you check other instances of `@keyframes` animation in this project and make sure your naming follows guidelines?